packages/s3/test/adapter.test.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  isObjectSession,
+  S3Client,
+  S3Storage,
+  S3StorageClient,
+} from '../adapter.ts';
+import { lazySession } from 'https://lib.deno.dev/x/grammy@1.x/mod.ts';
+import { createBot, createMessage } from '../../../libs/utils/src/mod.ts';
+import { assertSpyCall, spy } from 'jsr:@std/testing/mock';
+import { assert } from 'jsr:@std/assert';
```

Comment:
Is it deno related imports?

Reply:
Yes, is that a problem? wo you prefer an `imports` field in `deno.json`?

Reply:
> Yes, is that a problem? wo you prefer an `imports` field in `deno.json`?

I guess so for std'ed deps.

Reply:
:heavy_check_mark: